test/fixtures/poro.rb

```diff
@@ -258,4 +260,30 @@ def maker
   cache only: [:id]
   attributes :id
 end
+
+class TweetSerializer < ActiveModel::Serializer
+  attributes :id, :body, :date
+
+  belongs_to :author do
+    attributes :id
+  end
+
+  has_many :shares do
+    attributes :id, :platform
+
+    belongs_to :author do
+      attributes :id, :name
```

Comment:
I guess, a note for later - without sideloading, it's possible a have a very redundantly bloated resulting json